railties/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Make all rails commands work in engine.
+
+    By default, commands that require rails application are executed against
+    dummy testing application. If you want to execute a command against a real
+    application, define `APP_PATH` of real application to `bin/rails` script.
```

Comment:
I find this tough to understand. Why don't we mention the commands that couldn't be run before and why they now can (i.e. we link the engine to the dummy app via `APP_PATH`?

I think we should leave the override mention to an updated engine guide in another PR 😊